src/repository/default.rs

```diff
@@ -439,6 +439,51 @@ impl Repository for $repo {
         Ok(functions)
     }
 
+
+    async fn drop_function(
+        &self,
+        database_id: DatabaseId,
+        if_exists: bool,
+        func_names: &[String],
+    ) -> Result<(), Error> {
+
+        for function_name in func_names.iter() {
+            // Construct the SQL DELETE statement
+            let query =
+                format!(
+                    r#"
+                    DELETE FROM "function"
+                    WHERE database_id = $1 AND name = $2
+                    RETURNING id;
```

Comment:
You can delete all functions with one SQL statement: `DELETE FROM ... AND name IN (...). To construct that query you can use the sqlx query builder (see for example `get_all_table_versions`).

Reply:
Good idea, did it that way!